util/build_configs/setenv-example-3.bash

```diff
@@ -97,20 +97,25 @@ if [ -z "$BUILD_CONFIGS_CALLBACK" ]; then
     $cwd/build_configs.py -p $dry_run $verbose -s $cwd/$setenv -l "$project.runtime.log" \
         --target-compiler=$compilers --comm=$comms --launcher=$launchers
 
-        # The following 2x2x2 matrix of Chapel runtime configurations are defined by the
-        # arguments passed to build_configs.py, above. All eight runtimes will be built.
         # NOTE: "--target-compiler" values shown above will be discarded by the setenv callback.
 
-        # CHPL_TARGET_COMPILER  CHPL_COMM   CHPL_LAUNCHER
-        # --------------------  ---------   -------------
-        # cray-prgenv-gnu       none        none
-        # cray-prgenv-gnu       none        slurm-srun
-        # cray-prgenv-gnu       ugni        none
-        # cray-prgenv-gnu       ugni        slurm-srun
-        # cray-prgenv-intel     none        none
-        # cray-prgenv-intel     none        slurm-srun
-        # cray-prgenv-intel     ugni        none
-        # cray-prgenv-intel     ugni        slurm-srun
+        # The following 2x2x2 matrix of Chapel runtime configurations are defined by the
+        # arguments passed to build_configs.py, above.
+
+        # CHPL_TARGET_COMPILER  CHPL_COMM   CHPL_LAUNCHER   Make Chapel?
+        # --------------------  ---------   -------------   -----------
+        # cray-prgenv-gnu       none        none            Yes
+        # cray-prgenv-gnu       none        slurm-srun      No (skip)
```

Comment:
comm=none with a Chapel launcher is still worthwhile.  (Chapel provides its own launcher so that users don't have to know the details of how to launch programs on their systems, and this is just as valuable for single-node comm=none programs as it is for multi-node ones.)

The only config that doesn't make sense is comm=ugni with launcher=none; that one is known not to work because the ugni comm layer depends on some settings that are done by the Chapel launchers.